### PR TITLE
Feature/item tooltip

### PR DESCRIPTION
**Pull request by samuelWilliams99**
image
Adds a tooltip for items, items now support a description panel

### CODE REVIEW

File: gamemode/modules/inventory/cl_itemtooltip.lua [hunk 1]
```diff
@@ -0,0 +1,144 @@
+RVR.Inventory = RVR.Inventory or {}
+local inv = RVR.Inventory
+
+local PANEL = {}
+
+local h = ScrH() * 0.12
+local w = h * ( 921 / 261 ) * 1.6
```
Comment (overlordbleck) [inline gamemode/modules/inventory/cl_itemtooltip.lua]: what are these magic numbers

**Review by overlordbleck — COMMENTED** [review]
Not much to say with the code of this one but the design; I think we could go with a more vertical approach because if we put a description it will just be the entire width of that box. In my opinion that might look weird but I would love for you to convince me otherwise if you like this design.

File: gamemode/modules/inventory/cl_itemtooltip.lua (hunk 1 above)
Comment (samuelWilliams99) [reply gamemode/modules/inventory/cl_itemtooltip.lua]: I removed 2 of them, the other two are basically unavoidable though

**Review by ghost — APPROVED** [review]
I would add some TODOs at the items descriptions (Currently \<placeholder\>) but otherwise looks good

**Comment by samuelWilliams99** [comment]
They're changed in the food branch, but yeah will do

**Pull request merged**